ouroboros-consensus/src/Ouroboros/Consensus/Network/NodeToClient.hs

```diff
@@ -187,7 +187,7 @@ defaultCodecs ccfg version networkVersion = Codecs {
 
     , cStateQueryCodec =
         codecLocalStateQuery
-          (networkVersion >= NodeToClientV_8)
+          (networkVersion >= NodeToClientV_9)
```

Comment:
Looking at the definition of `codecLocalStateQuery`, I'm not sure that this guard should change. What do you think?

Reply:
Yes, good catch! I'll revert this change.